spec.bs

```diff
@@ -216,8 +209,15 @@ Issue: Ensure errors are of an appropriate type, e.g. {{InvalidAccessError}} is
 Exposing to global scopes {#exposing}
 =====================================
 
-To expose this API to a global scope, a [=read only=] [=attribute=] `privateAggregation`
-of type {{PrivateAggregation}} should be exposed on the global scope.
+To expose this API to a global scope, a [=read only=] [=attribute=]
+`privateAggregation` of type {{PrivateAggregation}} should be exposed on the
+global scope. Its [=getter steps=] should be set to the [=get the
+privateAggregation=] steps given [=this=].
+
+Each global scope should set the [=PrivateAggregation/allowed to use=] for the
+{{PrivateAggregation}} object it exposes based on whether the
```

Comment:
Maybe can rephrase to "based on whether the relevant document is allowed to use the "private-aggregation" policy control feature" and link properly?

Reply:
Had a go along these lines